jenkins-examples/pipeline-examples/declarative-examples/maven-jib-example/Jenkinsfile

```diff
@@ -0,0 +1,66 @@
+pipeline {
+    agent any
+    stages {
+        stage ('Clone') {
+            steps {
+                git branch: 'master', url: "https://github.com/jfrog/project-examples.git"
+            }
+        }
+
+        stage ('Artifactory configuration') {
+            steps {
+                jfrogInstance (
+                    id: "JFROG_INSTANCE",
+                    url: SERVER_URL,
```

Comment:
Notice the general instructions for all examples in README - https://github.com/jfrog/project-examples/tree/master/jenkins-examples/pipeline-examples

Step 1 - Configure Artifactory Server for the Examples
All examples use an Artifactory server defined in Jenkins' Configuration.

Find or configure an Artifactory server in Jenkins through Manage Jenkins, Configure System. You'll need its Server ID.
I suggest that instead for configuring the URL in the example, we stay consistent with the examples' instructions. The same goes for the scripted example.